### ISSUE TITLE
typeahead features in ComboBox

### ISSUE THREAD

**Issue by Tropix126** [Tropix126/fluent-svelte#36]
the native implementation doesn't have this, but I think it'll be beneficial for accessibility and would just be a nice feature to have.

**Comment by Tropix126** [comment]
might also apply to MenuFlyout
